### PR TITLE
add support for static external IPs

### PR DESCRIPTION
**Pull request by mvyletel**
It is little bit hack-ish but it does the trick.

### CODE REVIEW

File: modules/umig/main.tf [hunk 1]
```diff
@@ -51,6 +51,13 @@ resource "google_compute_instance_from_template" "compute_instance" {
     subnetwork         = var.subnetwork
     subnetwork_project = var.subnetwork_project
     network_ip         = length(var.static_ips) == 0 ? "" : element(local.static_ips, count.index)
+
+    dynamic "access_config" {
+      for_each = list(element(var.static_external_ips, count.index))
```
Comment (aaron-lane) [inline modules/umig/main.tf]: Unfortunately, this approach does not work when `var.static_external_ips` is empty:

         Error: Error in function call
       
         on ../../../../modules/umig/main.tf line 56, in resource "google_compute_instance_from_template" "compute_instance":
         56:       for_each = list(element(var.static_external_ips, count.index))
           |----------------
           | count.index is 0
           | var.static_external_ips is empty list of string
       
       Call to function "element" failed: cannot use element function with an empty
       list.

Refer to how [network_ip](https://github.com/terraform-google-modules/terraform-google-vm/pull/53/files#diff-f4b43b1c49940f80bc7c1b08e83fb7aeR53) is configured for a suggestion of how to fix this.

**Comment by aaron-lane** [comment]
@mvyletel also, please rebase your branch against master.

File: modules/umig/main.tf (hunk 1 above)
Comment (mvyletel) [reply modules/umig/main.tf]: oh you're right. weirdly enough it worked for my while testing it locally... anyways fixed

**Comment by mvyletel** [comment]
hm still failing :\ but I can't see the error so can you please share why it failed @aaron-lane ? thanks

**Comment by omazin** [comment]
Think it's better to create `access_config` variable, as it's done for `instance_template` submodule https://github.com/terraform-google-modules/terraform-google-vm/pull/43

**Pull request closed without merging**